packages/app/src/specs/InlineSpecListHeader.vue

```diff
@@ -110,7 +111,7 @@ const input = ref<HTMLInputElement>()
 const onInput = (e: Event) => {
   const value = (e.target as HTMLInputElement).value
 
-  emit('update:specFilterModel', value)
+  emit('update:specFilterModel', getPathForPlatform(value))
```

Comment:
We could move the `getPathForPlatform` into the `useSpecFilter` function, that way we could consolidate the platform handling in one spot.